src/Tizen.NUI/src/public/BaseComponents/TextEditorEvent.cs

```diff
@@ -280,5 +334,66 @@ public TextEditor TextEditor
                 }
             }
         }
+
+        /// <summary>
+        /// The AnchorChanged event arguments.
+        /// </summary>
+        /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class AnchorTouchedEventArgs : EventArgs
+        {
+            private TextEditor textEditor;
+            private string href;
+            private uint hrefLength;
+
+            /// <summary>
+            /// TextEditor.
+            /// </summary>
+            /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public TextEditor TextEditor
+            {
+                get
+                {
+                    return textEditor;
+                }
+                set
+                {
+                    textEditor = value;
+                }
+            }
+            /// <summary>
+            /// Anchor href.
+            /// </summary>
+            /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public string Href
+            {
+                get
+                {
+                    return href;
+                }
+                set
+                {
+                    href = value;
+                }
+            }
+            /// <summary>
+            /// Anchor href length.
+            /// </summary>
+            /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public uint HrefLength
```

Comment:
We don't need length, because string has length property

Reply:
I agree. It seems there is no need to provide HrefLength property in NUI.